res/loc/po/zh-CN.po

```diff
@@ -288,39 +288,39 @@ msgstr "字节"
 #.
 #. *Short* version of the kilobyte size suffix
 msgid "KB"
-msgstr ""
+msgstr "KB"
```

Comment:
This is unneeded. If a translation is the same as English, then English will be used. Which means that, if "KB" is not translated for Chinese, then it will be correctly be displayed as "KB".